### PR TITLE
 make the alias unified

### LINKED ISSUE #60192: The blended alias in import are so confused, can we modify this to be unified?
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line: 
>
> /kind feature

**What happened**:

- I found that the the blended alias in import are so confused, can we modify this to be unified?

**What you expected to happen**:

- The alias of the same import package in different packages are unified.

### PR DESCRIPTION
**Pull request by lcfang**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #60192 

**Special notes for your reviewer**:

**Release note**: 

```release-note
NONE
```

### CODE REVIEW

**Comment by bsalamat** [comment]
Please put "NONE" in the release note section of the PR.

/lgtm
for the changes in the scheduler code.

**Comment by lcfang** [comment]
Hi @bsalamat , I just rebased my commits, can you add the label again? Thanks.

**Comment by lcfang** [comment]
/assign @liggitt @wojtek-t 
the aliases in import are non-standard, I think we should make the standard and make them unified.

File: pkg/apis/core/v1/conversion_test.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ import (
 	"k8s.io/kubernetes/pkg/api/legacyscheme"
 	"k8s.io/kubernetes/pkg/apis/core"
 	corefuzzer "k8s.io/kubernetes/pkg/apis/core/fuzzer"
-	corev1 "k8s.io/kubernetes/pkg/apis/core/v1"
+	k8s_api_v1 "k8s.io/kubernetes/pkg/apis/core/v1"
```
Comment (liggitt) [inline pkg/apis/core/v1/conversion_test.go]: underscored aliases read strangely to me... I prefer the existing aliases as-is

File: pkg/controller/disruption/disruption_test.go [hunk 2]
```diff
@@ -23,7 +23,7 @@ import (
 	"testing"
 	"time"
 
-	apps "k8s.io/api/apps/v1beta1"
+	appsv1beta1 "k8s.io/api/apps/v1beta1"
```
Comment (liggitt) [inline pkg/controller/disruption/disruption_test.go]: +1

File: pkg/apis/core/v1/conversion_test.go (hunk 1 above)
Comment (lcfang) [reply pkg/apis/core/v1/conversion_test.go]: Hi @liggitt , thanks for your attention. 
the aliases in kubernetes are so chaotic, I think we can have some standard, not casual.

Comment (liggitt) [reply pkg/apis/core/v1/conversion_test.go]: `corev1` maps to `.../core/v1` better than `k8s_api_v1`

Comment (liggitt) [reply pkg/apis/core/v1/conversion_test.go]: essentially, the algorithm is if the package name (in this case, `v1`) is ambiguous, include the next package level up as well (`corev1`)

Comment (wojtek-t) [reply pkg/apis/core/v1/conversion_test.go]: +1 to what @liggitt wrote - we are trying to use the convention of using the shortest suffx (in terms of number of elements of the path) that makes it unique. So corev1 here is what we want.

**Pull request closed without merging**